projects/python/perception/object_detection_2d/yolov5/inference_tutorial.ipynb

```diff
@@ -0,0 +1,305 @@
I86nHr1jVu3bqFRpP1uuifPMVaifqROFbPdDoHbxj0BzgnqgnTitM4DIXe1DSGNEskm6ypcd6RJplQdb3DOE+cpJiyZLVas1iuWS6X1HVD3dREYUieZxcw0TCMMAZOT0VU65WmaWrKdUlZ19LGHg0IwxClA2aLOc4a4jghTjLqtmG9WqFQxFlGax3eWAIdUtWGF7/0Cov5XKhTsQROTM+mnY9O3FTT+RxnxfNVloUMaPd32N7YojWWBwfHvH7nAev1mqZpGfQH1HXNg8MTkjhme3cbbw1n0xlHx2ecTudYJxSsOJJGzM7WBoPBQDp5Uch4NGQ86DGejNE4lssFDw8OOJsuKKuKphG5VtMYHJ4oTtDK4wMvWALnqDuGZNjrSQfWNDzxxHWeuHmDNImpStGZnpycsLW9iTW+M7E6VkXFpz7zee4/OMAay5NP3iDJe9RNQ5rnDLpd7+RsShgo0iynl+R46ygvRi/5V3Qdv6UXmbUO4yHPE6IwYjIec2lni929LZ57/hm2tzeJ4oAwUCRJhkLC6+IkEegLsF6tpa1cW4IwpCzLLs0+lgRHHJqAOE2p6xrvDc4pjHWcnZ1grSNJUgaDvnAwlEIHEXk/ZTabSWJI3VK3hrI2nJ7NKMqSLEsZtAZPQJrJHbqq666lbnHekkYJe3vbZHlO09b0+32cdd2bPqNpWqqypm4bTCMD8DDQWO+wrWNVlqKqbxqapqVpRfOX5xlKC6u+bhrCIJCzT5wQBCGHBwcATCYjitrym597kcVizcn0DEVA20qw4P2HR938ThoTvPxa57FTXeh7QBJFjEZ9elkGeDYnQ5IOmLM1mbC7v01TlQxHA+jwDBbFcdfMsZ0fTmnJXTu3GwVBgIpBq4gojrrzstyYXnj+3bznXc/jrKEsKvqjgdhgypKmMZTrUgIaTcNsvpBg9yCkbR2f+ezLpFnCoN/j6PCER4+OJT0mCnni1g1uXDdcurRL7GNUoLnzpde4f/8Bh4fHb/o6fkvLqq5evuz/5l/96zz93FP08pR+v4dXAhr11nW5xWIjMcawWC6wTrjrpm1YrQuiKJLUFuNEEuUB5SXSRymaqhWbe9PiOohLaxzT+RKtAyaTEeNhThJHoiTPEpyTO6VpTTc0NjSNZTpfMFssuuAE+flxEItq3DsB1bSGOA7p91KyTHLBrHGs1ktBk4XiTD4P0VNdoqW1BmMsTdtQ1S1N1XRjDUXT1F2pKM2NpmkwrfDjrTGEWpOmKVGWUndnwbxbFGVVc3JySlE2Fw7mpmnEMd2Vbs4Kw0NrwQXoQBOHIdeuXGJzPODy5T2GwxFploG35Ll042ZnZ2S9lNl0weHJKQ/uP8C0jrKqaZqW0bgP6DdypwOhUHnvBKrqBDturWC7A60Z9fu8813Pk0QBD+8/xFsIk5BHjw+5c/cBTWvYnGzw5FM3aE3Lw4ePWa5LWuN4+OiAxWJJGGqSKCIKJNQ9SRJ5rp2sLdABs9kM1zFFojDiJ3/mX3E2Pfvq0y6+44W3+Z/6Zz/CermQlPtQfFXOIwiyQFNVJbOzqagwnKcoC0n2CGTqv14VNG3LclGAEqR0UTRM54tOjtSwXhci4A0DsjSVlrQTNYdS4HEkcUKWpaLcbmoUIcNB1uUKG9rWyXmky9tKswwdhCgEVCrE4wi866RUQQf5fCPlxTrJefZe4DlagbNyUYJQzXSg0Tqgrluxr7SieDfOfhkDv0VpjTUSJphE8UXIXxgodna2L+hQ09mMprXUddPRojzWdDtJl/oSBJD3UvJcoKXDwYDNyYgnb1wFBTt7e5TrglVR4Kzj9ut3qTvEgHWe5WJF2VGDm9aQxCFZJqOA9aoEZB7qgTCO8XjCIOyAOkLIbk2Ld5YQ6Pf7pGnCfDZntSrw3tG0HWMkjFivluDPF7ORfLFuwXgkZD0INKPhgDCM6OU5VVkRaMViuSRJBWSkgoD1egUofuEXf4bZ7M0JhN/S5aL3nnK9ZjSZ4KxhvV4TZylZHHN4cCjotiiGIMAby9HRIXVjumSTklVRslyXzGYLzk6nFGXNbL6SVJfuMO0c5HnKxtZmd+EJ6HQ8GWCdxdk3BMOz2Rzd2URaYzidLtCBJs8zvNIEoWJjc4LyjnVRsi5KrHPEHR1XzIeWPMsYjwfkaQyIUXu5WlPUFdaeC14dddtQ1Yaykk5bEATkWQq+uSixkiyhrGvaRuJv6W5A4DEY4lAU7+ezpTAKWa7W0un0HtMx4s8d4VoHjIZ90VUCl/f3mIz7JHlKFMeMxmOpElYrgiTCNIbDoxOOj0+YLeacns7wXtAGTbdw+71ckmC8k7xv5zk9maMC6dAGWlE3ouapyoowkAG05KDJ64+CMIgwdc3J2ZzGNF1QoutuMo6m7rAIzpImGct1SRCF6DCSc7WGOIxpWnkfUDJor8uCpqoZDIekSdLJ2yANI/Zu3CQIFB//xJs/l72lF5nWAUmWMV8sCIOI3mBAW9csFgviLMUt19x7/TbWK3qDIWmeU7dzjk7OuHvvIWezOfNVwWJR4J1Y2JMkxltHGAX0s57EKXWdtX6/h/eOum45OZnR62UkaULbGJqmpNfrXVhP4iSVpEhryLIUBaRJKlDUsqCuGwaD3oVFJdAQZakwD5Xi5GxOliY422Jay7qqO2oWFKZ5I661g5xqrRgO++RpQpZlxEnMdLbgbDoHJDzBA2EQY62VRFJrCXWADgOatqGua9arovt5Hf7NG9H46YAb16/Qy1Nu3LzKjauXBAOXxtRlzd7lS5ydnrJel8yqitl0yenp50hSGVobZ2iNWHJM2+KcsCG9l9eorivhQTpLVdUkaUqaZ7RGAD8qFJ5JXVdoD0EU0LaNdBnbRnSiOqCqG6xzgmJw7kI2poOA1pg3IKQdLQukixxFEVXdUNUNzooJtKlqrl3Z59q1y/QHfc5OZxwdnXQlfUKgFcPxkLPTM5q6ftPX8Vt6kZ2XQhrFfHrGdOoIo4Q4CqmKkjhNeeLZZ5ieTT
```

Comment:
Same as previous comment (something goes wrong when i add suggestion here :confused:).

Reply:
This comment is meant for adding line_thickness again, for some reason it shows up on a wrong line for me.

Reply:
fixed in [ab30d52](https://github.com/opendr-eu/opendr/pull/360/commits/ab30d52ae46ff4fa2c3cc4bac54909302ad76f8a)